wls-gui-wahllokalsystem/src/components/ergebnisermittlung/OBW/stapelC/TheOBWStapelCErfassungCard.vue

```diff
@@ -90,24 +122,64 @@ const stapelCErgebnisseOrdereByNumIndex = computed(() => {
 });
 
 const isFormValid = ref<boolean | null>(null);
```

Comment:
```suggestion
const isRowsFormValid = ref<boolean | null>(null);
```
da es zwei forms in diesem file gibt finde ich den namen zu generisch

Reply:
applied with e8601dc2e31a06a833690e181d5abdd4f6587b73